README.md

```diff
@@ -270,6 +270,7 @@ A public list of companies that are known to use the Robot Operating System (ROS
 | [Niryo](https://niryo.com) |Design industrial robotic solutions composed of robot arms, vision, artificial intelligence and cloud computing. The Ned2[^59], 6-axis collaborative robot designed for Education. Ned ros stack (see [`ned_ros`](https://github.com/NiryoRobotics/ned_ros)). | 2016 | 
 | [Nissan](https://www.linkedin.com/company/nissan-motor-corporation) | Global car manufacturer. Looking for ROS talent[^260] in their Alliance Innovation Lab – Silicon Valley (AIL-SV) | 1933 |
 | [nLink mobile robotics](https://www.nlinkrobotics.com/) | Developing mobile robotic systems. Helps to realize robotics [visions](https://www.nlinkrobotics.com/tailored-robotic-concepts), and robotize repetitive and dangerous job tasks. With his [Cloud Robotics Software](https://www.nlinkrobotics.com/cloud-robotic-software) help robots fleet to stay synchronized and productive at all times [^121]. | 2012 |
+| [NODE Robotics](https://node-robotics.com/) | NODE offers plug and play software solutions for autonomous intralogistics. The focus is on driverless transport vehicles and mobile robots, which are turned into multi-vendor autonomous and [`collaborative fleets`](https://www.youtube.com/watch?v=N3CZEDsUADo) by their software solutions. NODE provides SW solutions for industrial-grade autonomous navigation as well as (multi-vendor) [`fleet management`](https://www.youtube.com/watch?v=BBaFcIEOCYo) & [`coordination`](https://www.youtube.com/watch?v=o9RGpZDgPXQ).| 2020 |
```

Comment:
```suggestion
| [NODE Robotics](https://node-robotics.com/) | NODE offers plug and play software solutions for autonomous intralogistics. Have been using ROS as middleware and toolkit since day one to build NODE.OS, a plug-and-play, industry-ready, and hardware-agnostic software stack for intralogistics applications[^315]. | 2020 |
```